### PR TITLE
Issue 11 form submission

### PR DESCRIPTION
**Pull request by danigirl329**

### CODE REVIEW

**Comment by wonderwoman13** [comment]
@danigirl329 I added some code to add the cookie and repair the controller actions. Before I tie this up, I am getting an array back from the Anecdote form (see TODO on the AnecdotesController). Can you please take a look to see why that is happening?

File: app/controllers/anecdotes_controller.rb [hunk 1]
```diff
@@ -1,17 +1,12 @@
 class AnecdotesController < ApplicationController
 
   def create
-    @anecdote  = Anecdote.create(anecdote_params)
+    #TODO: why am I getting an array from the Frontend? Can this be a string field?
```
Comment (wonderwoman13) [inline app/controllers/anecdotes_controller.rb]: @danigirl329 here is where I need some help, please.

**Comment by wonderwoman13** [comment]
p.s. I know the tests are failing, it is the AnecdotesController. Once we fix that up, should work great.

**Comment by danigirl329** [comment]
Tests are failing because no cookies exist :)

**Pull request merged**